### PR TITLE
430 slow starting consumers monitoring

### PR DESCRIPTION
**Pull request by pawelszymczyk**

### CODE REVIEW

**Comment by chemicL** [comment]
:+1:

File: hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/ConsumersSupervisor.java [hunk 1]
```diff
@@ -204,8 +206,10 @@ private void createAndExecuteConsumer(Subscription subscription) {
         logger.info("Creating consumer for {}", subscription.getId());
         try {
             Consumer consumer = consumerFactory.createConsumer(subscription);
+            logger.debug("Created consumer for {}", subscription.getId());
             consumerHolder.add(subscription.getTopicName(), subscription.getName(), consumer);
             executor.execute(consumer);
+            logger.debug("Consumer for {} is created and executed ", subscription.getId());
```
Comment (adamdubiel) [inline hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/ConsumersSupervisor.java]: *was created ;p

**Comment by adamdubiel** [comment]
:+1:

File: hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/ConsumersSupervisor.java (hunk 1 above)
Comment (pawelszymczyk) [inline hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/ConsumersSupervisor.java]: done

**Pull request merged**